pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/documentation/CommentRequiredRule.java

```diff
@@ -158,7 +158,7 @@ public Object visit(ASTConstructorDeclaration decl, Object data) {
 
     @Override
     public Object visit(ASTMethodDeclaration decl, Object data) {
-        if (isAnnotatedOverride(decl)) {
+        if (decl.isOverridden()) {
```

Comment:
This looks like a good improvement for CommentRequiredRule! :+1: 
Ok, so these are the removed violations.